### PR TITLE
LPS-87919 Add unit tests

### PR DESCRIPTION
**Pull request by leoadb**

### CODE REVIEW

**Comment by leoadb** [comment]
ci:test:sf

**Comment by jeyvison** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/data-engine/data-engine-web/src/main/java/com/liferay/data/engine/web/internal/servlet/data/fetcher/DESaveDataDefinitionDataFetcher.java [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+	extends DEBaseDataDefinitionDataFetcher
+	implements DataFetcher<SaveDataDefinitionType> {
+
+	@Override
+	public SaveDataDefinitionType get(DataFetchingEnvironment environment) {
+		long userId = GetterUtil.getLong(environment.getArgument("userId"));
+		long groupId = GetterUtil.getLong(environment.getArgument("groupId"));
+
+		Map<String, Object> dataDefinitionInput = environment.getArgument(
+			"dataDefinitionInput");
```
Comment (leoadb) [inline modules/apps/data-engine/data-engine-web/src/main/java/com/liferay/data/engine/web/internal/servlet/data/fetcher/DESaveDataDefinitionDataFetcher.java]: @jeyvison Let's change this to "dataDefinition", without "Input"

File: modules/apps/data-engine/data-engine-web/src/test/java/com/liferay/data/engine/web/internal/servlet/data/fetcher/DESaveDataDefinitionDataFetcherTest.java [hunk 2]
```diff
@@ -0,0 +1,1010 @@
...
+			{
+				put("descriptions", descriptions);
+				put("fields", fields);
+				put("names", names);
+				put("storageType", "xml");
+			}
+		};
+
+		when(
+			dataFetchingEnvironment.getArgument("dataDefinitionInput")
```
Comment (leoadb) [inline modules/apps/data-engine/data-engine-web/src/test/java/com/liferay/data/engine/web/internal/servlet/data/fetcher/DESaveDataDefinitionDataFetcherTest.java]: @jeyvison Please replace this by "dataDefinition" according to my past suggestion. Thanks!

**Comment by jeyvison** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/65897. [See changes here.](https://github.com/jeyvison/liferay-portal/compare/f84985098a1c~1...f84985098a1c) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jeyvison** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/66018. [See changes here.](https://github.com/jeyvison/liferay-portal/compare/e688a79a3518~1...e688a79a3518) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jeyvison** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/66031. [See changes here.](https://github.com/jeyvison/liferay-portal/compare/eca4ba184d0c~1...eca4ba184d0c) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jeyvison** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/66053. [See changes here.](https://github.com/jeyvison/liferay-portal/compare/f22ef5a0f9b6~1...f22ef5a0f9b6) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**